sale_stock_picking_note/views/res_partner_view.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Moduon Team S.L.
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0) -->
+<odoo>
+    <record id="view_partner_form_inherit_partner_picking_note" model="ir.ui.view">
+        <field name="name">Partner Picking Note</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='sale']" position="inside">
+                <field name="picking_note" placeholder="Picking notes..." />
```

Comment:
question: Maybe this should be under the stock user group? I mean, isn't it a problem that any user in the organization can add a default note to any contact?

Reply:
At least ensure Sale User group (don't remember if the tab has this group)

This will be propagated from Sales to Pickings in all cases.

Reply:
> question: Maybe this should be under the stock user group? I mean, isn't it a problem that any user in the organization can add a default note to any contact?


> At least ensure Sale User group (don't remember if the tab has this group)
> 
> This will be propagated from Sales to Pickings in all cases.

Added sales: own documents and stock user to fields